schedule/jeos/sle/rpi/jeos-main.yaml

```diff
@@ -11,6 +11,7 @@ schedule:
   - jeos/diskusage
   - jeos/build_key
   - console/prjconf_excluded_rpms
+  - console/journal_check
```

Comment:
We have `journal_check` already at [jeos-filesystem](https://openqa.suse.de/tests/8675790#step/journal_check/7) test. Do we need it on another test that doesn't run on real hw?

Reply:
yes, it would be nice to see if we catch any of those [bsc#1198534](https://bugzilla.opensuse.org/show_bug.cgi?id=1198534) [bsc#1198524](https://bugzilla.opensuse.org/show_bug.cgi?id=1198524) [bsc#1198527](https://bugzilla.opensuse.org/show_bug.cgi?id=1198527)

Reply:
These bugs seem to make sense to check for on real hw. And I think that we should run `journal_check` on real hw.
But that is done in the other yaml file.

THIS yaml file is about `jeos-main` which runs in a qemu VM. But we already run `journal_check` in  [jeos-filesystem](https://openqa.suse.de/tests/8675790#step/journal_check/7) which runs in a qemu VM as well.

Reply:
ok, I just though it doesn't harm having it in main by default too as in other tests, but ok.